mtp_noms_ops/apps/security/utils.py

```diff
@@ -67,8 +67,8 @@ def convert(obj):
                         new_value = timezone.localtime(new_value)
                     obj[field] = new_value
                     break
-                except (ValueError, TypeError) as e:
-                    logger.warning(e)
+                except (ValueError, TypeError):
+                    logger.exception('Failed to convert date fields in object list')
```

Comment:
upgrading warning to exception because it indicates a mismatch between what api returns and what noms-ops expects